web/server/codechecker_server/api/report_server.py

```diff
@@ -2816,10 +2832,11 @@ def get_analyzer_name(report):
                         elif status == 'intentional':
                             rw_status = ttypes.ReviewStatus.INTENTIONAL
 
-                        self._setReviewStatus(report_id,
+                        self._setReviewStatus(session,
+                                              report_id,
                                               rw_status,
                                               src_comment_data[0]['message'],
-                                              session)
+                                              detected_at)
```

Comment:
The `run_history_time` variable should be used here because the `detected_at` variable will be updated in some cases (`detected_at = old_report.detected_at`). See the codes above.